docs/src/pages/demos/steppers/HorizontalLinearStepper.js

```diff
@@ -75,7 +75,7 @@ class HorizontalLinearStepper extends React.Component {
     }
 
     this.setState(state => {
-      const skipped = new Set(state.skipped.values());
+      const skipped = new Set(state.skipped);
```

Comment:
that is still `new Set(iterable)` which is not supported in IE11: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set#Browser_compatibility

I guess the underlying problem is that IE 11 does not implement the iterator protocol via `@@iterator`.

Reply:
It is. Either IE 11 fast tracks this somehow and not considers this an iterable or the MDN data is outdated.

Reply:
Just tested it in the console and it doesn't. 

The hole skipped logic is currently dead code since it is only responsible for https://github.com/mui-org/material-ui/blob/ce81eb82cb616c1585c98bf882877ab5aaeb4c4b/docs/src/pages/demos/steppers/HorizontalLinearStepper.js#L111-L113
but steps are not completed by default https://github.com/mui-org/material-ui/blob/ce81eb82cb616c1585c98bf882877ab5aaeb4c4b/packages/material-ui/src/Step/Step.js#L153

Reply:
Ok, it doesn't work 😢 
```
var s1 = new Set()
s1.add('a')
console.log(s1.has('a'))
var s2 = new Set(s2)
console.log(s2.has('a'))
```
outputs
```
true
false
```